spec/factories/facilities.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :facility do
-    sequence(:name) { |n| "Facility#{n}" }
+    sequence(:name, "AAAAAAAA") { |n| "Facility#{n}" }
```

Comment:
We're doing this enough that I wish we had something like `alphabetical_sequence(:name)`.